src/components/book.js

```diff
@@ -1,15 +1,24 @@
+/* eslint-disable no-unused-vars, jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, max-len */
```

Comment:
- [ ] Please `Linters` are there to help us stick to best practices, instead of disabling `eslint`, it will be nice to fix the issue.